app/serializers/supplejack_api/concept_serializer.rb

```diff
@@ -34,8 +33,11 @@ def include_context_fields!(hash)
       fields = hash.dup
       fields.shift
 
-      context = build_context(fields.keys)
-      hash['@context'] = context
+      if self.options[:inline_context] == "true"
+        hash['@context'] = build_context(fields.keys)
+      else
+        hash['@context'] = "#{self.options[:domain]}/schema"
```

Comment:
Can we have a route entry for `schema`? Like `get /schema, to: 'schemas#show', as: 'schema'` and use `schema_path` instead of concatenating `self.options[:domain]`.